server/data/quotes.json

```diff
@@ -23,6 +23,10 @@
     "author": "Amy Chua",
     "sentence": "Do you know what a foreign accent is? It’s a sign of bravery."
   },
+  {
+    "author": "Johann Wolfgang von Goethe",
```

Comment:
Please change the position of the quote alphabetically as suggested in the [contribution guide](https://github.com/jayehernandez/letra-extension/blob/master/docs/new_quote.md#-checklist).

Reply:
@Kvaibhav01 there is ambiguity about the alphabetic order, If I take a look at the previously added quotes, I don't find them ordered.
Are they ordered based on the Starting Letter of the sentence or the First Letter of the Author Name?

Reply:
Hey, they're ordered on the first letter of the author's name.

Reply:
Thanks for Clarifying, a quick suggestion, it should be updated in the Contribution Guide as well as in the PR template.